src/appleseed-max-impl/appleseedinteractive/interactiverenderercontroller.h

```diff
@@ -31,17 +31,30 @@
 // appleseed.renderer headers.
 #include "renderer/api/rendering.h"
 
+// appleseed.foundation headers.
+#include "foundation/platform/windows.h"    // include before 3ds Max headers
+
+// 3ds Max headers.
+#include <plugapi.h>
+
+// Forward declarations.
+class RendProgressCallback;
+
 class InteractiveRendererController
   : public renderer::DefaultRendererController
 {
   public:
-    InteractiveRendererController();
+    InteractiveRendererController(RendProgressCallback* progress_cb);
```

Comment:
Mark the constructor `explicit` to prevent implicit consutrction from a `RendProgressCallback`.